# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/midi/tests/browser.ini
# Commit: cfba299e5067
# Full Hash: cfba299e5067a4eab3f3b982374b0770970a3401
# Author: Gabriele Svelto <gsvelto@mozilla.com>
# Date: 2022-04-12 03:57:01
# Regressor Bug: 1758468
# File Overlap Count: 1
# Description:
#   Bug 1758468 - Use stable, anonymous IDs for Web MIDI ports r=padenot
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D142550
# ==============================================================================

diff -r c0eb987d0f76 -r cfba299e5067 dom/midi/tests/browser.ini
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/dom/midi/tests/browser.ini	Mon Apr 11 17:43:09 2022 +0000
@@ -0,0 +1,12 @@
+[DEFAULT]
+prefs =
+  dom.webmidi.enabled=true
+  midi.testing=true
+  midi.prompt.testing=true
+  media.navigator.permission.disabled=true
+
+[browser_stable_midi_port_ids.js]
+run-if = (os != 'android')
+support-files =
+  port_ids_page_1.html
+  port_ids_page_2.html